src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -193,6 +193,13 @@
                 .desc("Path to the bag config file")
                 .build());
 
+        configOptions.addOption(Option.builder("R")
+                .longOpt("repositoryRoot").argName("uri")
```

Comment:
argName("uri") is called twice here.

Reply:
I was following the model of the other options, where `argName` is called for both the long opt and short option.

Reply:
Odd, it doesn't seem like calling it twice would do anything unique here. _But_ I can't say I fully understand what Option.Builder is doing in the first place.